MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,452 @@
+        CommitMode="PropertyChanged">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+### Manual
+If the commit mode is Manual, the value should be committed manually by calling the [SfDataForm.Commit] method.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="Manual">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.Manual;
+{% endhighlight %}
+{% endtabs %}
+
+The following code commits the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+this.DataForm.Commit();
+{% endhighlight %}
+{% endtabs %}
+
+N> On manual commit call manual validation will be called to validate the properties before commit
+
+## Converter
+
+To show the original value in different format or as different value, use the [DataFormValueConverter] attribute.
+
+### Changing original value of the DataForm property value using converter
+
+Here, the original value is multiplied by 10 and shown in editor. While committing, it is divided by 10 and stored in the data object.
+
+{% tabs %}
+{% highlight c# %}
+public class ValueConverterExt : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {            
+        var amount = double.Parse(value.ToString());
+        return amount * 10;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var amount = double.Parse(value.ToString());
+        return amount / 10;
+    }
+}
+
+private double? amount = 1000;
+
+[DataFormValueConverter(typeof(ValueConverterExt))]
+
+public double? Amount { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Disable editing
+Editing DataForm can be disabled by setting the [IsReadOnly] property of the data form.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm" 
+        IsReadOnly="True">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.IsReadOnly = true;
+{% endhighlight %}
+{% endtabs %}
+
+The editing behavior can also be changed by setting the [IsReadOnly] property of the [DataFormItem].
```

Comment:
The particular editor can disabled  by setting the [IsReadOnly] property of the [DataFormItem].